pkg/api/client_runtime.go

```diff
@@ -27,16 +28,22 @@ import (
 )
 
 const (
-	// RegionPrefix is used when a Region is passed as part of the API config.
-	RegionPrefix = "/api/v1/regions/%s"
+	// DefaultPrefix is used as the base prefix for the API.
+	DefaultPrefix = "/api/v1"
```

Comment:
would it make sense to use [`client.DefaultBasePath`](https://github.com/elastic/cloud-sdk-go/blob/master/pkg/client/rest_client.go#L61) instead of declaring it again here?